src/controllers/controller.line.js

```diff
@@ -115,16 +115,19 @@ module.exports = DatasetController.extend({
 	updateElement: function(point, index, reset) {
 		var me = this;
 		var meta = me.getMeta();
-		var dataset = me.getDataset();
 		var datasetIndex = me.index;
-		var value = dataset.data[index];
+		var xScale = me._xScale;
+		var yScale = me._yScale;
 		var lineModel = meta.dataset._model;
+		var optStacked = yScale.options.stacked;
+		var stacked = (optStacked || (optStacked === undefined && me._cachedMeta.stack !== undefined));
```

Comment:
Maybe we could set `me._stacked` once in `update` so that we don't have to recalculate for each element?

Reply:
had to do it in 2 places, but done